src/nuclide_header.F90

```diff
@@ -539,6 +592,18 @@ subroutine nuclide_create_derived(this)
           this % sum_xs(t) % total(j:j+n-1) = this % sum_xs(t) % total(j:j+n-1) + &
                rx % xs(t) % value
 
+          ! Calculate nu-photon total cross section
```

Comment:
What we may end up needing to do is writing out those summed reactions to the HDF5 libraries.